### ISSUE TITLE
TextInput should be able to receive / forward the `readonly` attribute

### ISSUE THREAD

**Issue by davinotdavid** [thunderbird/services-ui#99]
Currently, the `TextInput` is able to receive `disabled` but not `readonly`. It would be very nice to have it for situations like the one below, where we still want to keep the "clicking the label auto-selects the text" property (which doesn't work if the input is disabled).

<img width="601" height="95" alt="Image" src="https://github.com/user-attachments/assets/200662e1-3391-477f-8dad-1d40d9785c57" />

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/readonly
